packages/autocomplete-js/src/requesters/getAlgoliaFacets.ts

```diff
@@ -0,0 +1,5 @@
+import { createAlgoliaRequester } from './createAlgoliaRequester';
+
+export const getAlgoliaFacets = createAlgoliaRequester({
+  transformResponse: (result) => result.facetHits,
```

Comment:
```suggestion
  transformResponse: (response) => response.facetHits,
```

(there's some more like this)

Reply:
See 6e42632d0310351822bb698a9000fe14f7acdcc1.